common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -0,0 +1,752 @@
+"""
+Bok choy acceptance and a11y tests for problem types in the LMS
+
+See also lettuce tests in lms/djangoapps/courseware/features/problems.feature
+"""
+import random
+import textwrap
+
+from abc import ABCMeta, abstractmethod
+from nose.plugins.attrib import attr
+from selenium.webdriver import ActionChains
+
+from capa.tests.response_xml_factory import (
```

Comment:
I don't know of other cases where we are doing this, but we were using this in the Lettuce tests and I didn't want to duplicate code just to have it in a different path.